### PR TITLE
Improve Gateway articles based on user feedback

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: It should be on Runtime Manager instead of on *the* Runtime Manager.

Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: It would be good to add a link here to actual configuration details as in https://docs.mulesoft.com/api-manager/2.x/org-credentials-config-mule3 and https://docs.mulesoft.com/api-manager/2.x/org-credentials-config-mule4. Take into account, for rxample, that for EU plane you need to also specify the base URL.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: This way of specifying the values only applies to Cloudhub.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: *the* API Manager.

Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: other ~kinds~ types of endpoints

Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: It should be clear that API Gateway can only be used with HTTP(s) endpoints for now.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc]: This functionality only applies to API Gateway, that means runtime 3.8.0 or later, or API Gateway 2.x.x.

Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc]: Not clear "configure API Gateway channel all its' communications"

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:12]: Corresponding "Configuring Organization Credentials" has more details.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: lower case for environment variables.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/configuring-an-api-gateway.adoc]: change one of the two "full" to a different word.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:43]: The path in the example correspond to deprecated API Gateway version, it would be better to use latest runtime 3.x or 4.x as an example.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc]: delete "you can"
no apostrophe on its (it's the possessive form, not the contraction for "it is")

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:37]: If you are using Mule 4, in addition to the mentioned parameters, usually, you would need something to disable HTTPs certificate validation, since many proxies do not have a valid certificate configured.

**Review by nahuel-dallavecchia — CHANGES_REQUESTED** [review]

**Review by Mulesoft-Mysti — APPROVED** [review]
A couple of grammar nits, otherwise looks great!
You should be able to merge after edits with this approval.
I didn't test the links.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: Dropping the `the`s for the API gateway and API Manager: 7e4d0b9

Regarding: 
> It should be clear that API Gateway can only be used with HTTP(s) endpoints for now

Should I drop mentions of Web Service Consumer?

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: Yes. This is under the **Manual Deployment to CloudHub** title.

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: I missed this one. 
See 3560c09

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc]: I rewrote this in 427d457

File: modules/ROOT/pages/configuring-an-api-gateway.adoc
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-an-api-gateway.adoc]: I would advise dropping the Web Service Consumer connector statement, since its not supported.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:12]: Can you add a link to it?

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:37]: Can you add the key anypoint.platform.enable_ssl_validation for the Mule 4 scenario?

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:43]: Can you update the path?

**Review by nahuel-dallavecchia — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:12]: This article is part of the API Manager 2.x. 
It looks like those instructions are valid for pre-crowd API Manager as well. I can bring those instructions back to 1.x and link to it.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (fermujica) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:37]: Is API Manager 1.x compatible with Mule 4? These changes are meant to quickly fix some "archived" labels from the 1.x docs.

Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:37]: Ah! If only for runtimes < Mule 4, then disregard comment.

File: modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/configuring-proxy-access-to-an-api.adoc:12]: Umm. The instructions are valid for Pre Crowd, except for the environments part. You may copy the article, removing any reference to environments and it should work. Considering that this is a Pre Crowd organization documentation, I'll leave it to you to decide wheter improve the doc or not.

**Review by nahuel-dallavecchia — APPROVED** [review]
Some pending minor stuff to fix, but considering it applies to pre Crowd type organizations, it's not critical.

**Pull request merged**
